modules/virt-example-bond-nncp.adoc

```diff
@@ -13,6 +13,8 @@ to the cluster.
 {VirtProductName} only supports the following bond modes:
 
 * mode=1 active-backup +
+* mode=2 balance-xor +
+* mode=4 802.3ad (lacp) +
```

Comment:
Please drop `(lacp)`. These right-hand values represent valid options in nmstate API.